src/components/content/content.scss

```diff
@@ -53,6 +53,13 @@ ion-content.js-scroll > .scroll-content {
   will-change: initial;
 }
 
+ion-content[no-scroll] > .scroll-content {
+  overflow-x: initial;
+  overflow-y: initial;
+  -webkit-overflow-scrolling: auto;
+  will-change: initial;
```

Comment:
If it is somewhere else, that is good. You can perhaps then unify both to:
```
ion-content.js-scroll > .scroll-content {
  position: relative;

  min-height: 100%;
}

ion-content.js-scroll > .scroll-content,
ion-content[no-scroll] > .scroll-content {
  overflow-x: initial;
  overflow-y: initial;
  -webkit-overflow-scrolling: auto;
  will-change: initial;
}
```

to remove the code clone, only if you think that is ok.